### ISSUE TITLE
Line break inserted before and after anchor tag

### ISSUE THREAD

**Issue by cayman1066** [danomatic/react-pdf-html#41]
Hello,

I am having an issue with **line breaks** appearing before and after an anchor tag passed as part of a string to the **HTML** component.

Key Packages:
- "@react-pdf/renderer": "^3.0.0",
- "react-pdf-html": "^1.1.14",
- "react": "^18.2.0",
- "react-dom": "^18.2.0"
    
Using @react-pdf/rendered **Link** works as expected, i.e. the anchor tag is placed inline.  However, when passing an html string with an anchor tag to **HTML** a line break gets inserted before and after the anchor tag.

**FILE: TestPDF.js**
```
import React from 'react';
import { Page, Text, View, Document, StyleSheet, Link } from '@react-pdf/renderer';
import Html from 'react-pdf-html';

const styles = StyleSheet.create({
	page: {
		flexDirection: 'column',
		padding: 16,
		backgroundColor: '#FFFFFF',
		fontSize: 12
	}
});

const htmlString = '<p>Link to <a style="display: inline-flex" href="//www.google.com">Google</a> using react-pdf-html.</p>';

const TestPDF = () => (
	<Document>
		<Page size="LETTER" style={styles.page}>
			<View>
				<Text>
					This is a <Link src="//www.google.com"><Text>link</Text></Link> to Google. And more
				</Text>
			</View>
			<View>
				<Html style={{ fontSize: 12 }}>{htmlString}</Html>
			</View>
		</Page>
	</Document>
);

export default TestPDF;

```

**FILE: App.js**
```
import { PDFViewer } from '@react-pdf/renderer';
import TestPDF from './TestPDF';

function App() {
  return (
    <PDFViewer style={{ width: '100%', height: 1200 }}>>
      <TestPDF />
    </PDFViewer>
  );
}

export default App;
```

**RESULT**
![image](https://user-images.githubusercontent.com/11344710/195137414-d29c606e-1e60-4721-b111-8dd647ade4b2.png)

Any suggestions for getting the anchor to appear inline using the <HTML /> component?

Thx!

**Comment by danomatic** [comment]
I think what’s happening is:

- `p` tag is being rendered as a View
- “link to “ is being rendered as a Text element
- The Link is rendered as a child of the `p` View, so it still has a flex/block level layout

I’ll have to figure out a good solution for this because it was caused as a result of fixing two other errors where Link and Image elements were being incorrectly rendered as inline. In the meantime, if you wrap the contents of the `p` tag in a `span` tag it would probably avoid  the issue.

**Comment by danomatic** [comment]
@cayman1066 this is fixed in the current version `v1.1.15`

**Comment by denimamab** [comment]
The bug is fixed but it introduced a new bug that the link is not clickable anymore
